src/components/SearchForm.vue

```diff
@@ -7,12 +7,14 @@
       placeholder="Type a dog breed..."
       ref="searchInput"
       @input="debounceSearch"
+      :disabled="isDisabled"
     />
   </form>
 </template>
 
 <script>
 export default {
+  props: ["isDisabled"],
```

Comment:
prop je dobr tud zdefinirat.. recimo ker type je, a je requested a ne, če ni requested pol določit default value https://v2.vuejs.org/v2/guide/components-props.html

Reply:
A se ponavadi vedno zdefinira prop, ali odvisno od primera?

Reply:
jaz jih vedno.. zato da je type check, da če je `isDisabled` boolean value, pol nemoreš dat notr array recimo.. to je dobra zaščita pol za kasnej ko buildaš projekt, da veš kaj si mislu in kaj pričakovat itd..